supabase-ws-flutter-mobile/lib/ui/signin/signin_screen.dart

```diff
@@ -23,26 +26,49 @@ class _SignInContentScreen extends StatelessWidget {
   Widget build(BuildContext context) =>
       BlocBuilder<SignInCubit, SignInBaseState>(
         builder: (context, state) => Scaffold(
-          appBar: AppBar(
-            title: Text(context.localizations.sign_in),
-          ),
-          body: Column(
-            mainAxisSize: MainAxisSize.min,
-            children: [
-              Expanded(child: _SignInForm()),
-              if (context.read<SignInCubit>().state.error.isNotEmpty)
-                Text(
-                  context.localizations
-                      .error(context.read<SignInCubit>().state.error),
-                ),
-              Padding(
-                padding: const EdgeInsets.symmetric(vertical: 30.0),
-                child: TextButton(
-                  onPressed: () => context.read<SignInCubit>().signIn(),
-                  child: Text(context.localizations.sign_in),
-                ),
+          body: SafeArea(
+            child: Padding(
+              padding: const EdgeInsets.all(16.0),
+              child: Column(
+                mainAxisSize: MainAxisSize.min,
+                children: [
+                  const SizedBox(
+                    height: 180,
+                  ),
+                  _SignInForm(),
+                  if (context.read<SignInCubit>().state.error.isNotEmpty)
+                    Text(
+                      context.localizations
+                          .error(context.read<SignInCubit>().state.error),
+                    ),
```

Comment:
how are you showing the errors? I'd show just an alert to be consistent

Reply:
we are going to update in sign in and sign up later